chardet/charsetprober.py

```diff
@@ -79,8 +78,8 @@ def filter_international_words(buf):
 
         This filter applies to all scripts which do not use English characters.
         """
-        filtered = BytesIO()
 
+        out = b''
```

Comment:
`bytes` are not mutable in Python, so switching this to using `bytes` and then concatenating to it with `+=` means creating lots of temporary strings.  `BytesIO` should be faster (although, feel free to prove me wrong).